lte/gateway/python/magma/enodebd/stats_manager.py

```diff
@@ -370,7 +370,7 @@ def _clear_stats(self) -> None:
         """
         logger.info('Clearing performance counter statistics')
         # Set all metrics to 0 if eNodeB not connected
-        for pm_name, metric in self.PM_FILE_TO_METRIC_MAP:
+        for pm_name, metric in self.PM_FILE_TO_METRIC_MAP.items():
```

Comment:
any of the stats unit tests covers this ?

Reply:
added an unit test that covers it